pkg/ingester/ingester_test.go

```diff
@@ -572,3 +575,21 @@ func BenchmarkIngesterPush(b *testing.B) {
 	}
 
 }
+
+func TestRemoveEmptyDir(t *testing.T) {
+
+	// remove dir that dne
+	require.NoError(t, removeEmptyDir(fmt.Sprintf("%v", rand.Int63())))
+
+	// remove empty dir
+	dir, err := ioutil.TempDir("", "TestRemoveEmptyDir")
+	require.NoError(t, err)
+	require.NoError(t, removeEmptyDir(dir))
+
+	// remove non-empty dir
+	dir, err = ioutil.TempDir("", "TestRemoveEmptyDir")
+	require.NoError(t, err)
+
+	ioutil.WriteFile(filepath.Join(dir, "tempfile"), []byte("hello world"), 0777)
+	require.NotNil(t, removeEmptyDir(dir))
```

Comment:
You can use `require.Error()` here, to signal more clearly the intention.

Reply:
Yea but that looks for a specific error, and I got lazy

Reply:
Oh you're right! Then your is better :)

Reply:
:)